Sources/StreamVideoSwiftUI/CallingViews/MicrophoneChecker.swift

```diff
@@ -22,20 +20,22 @@ public class MicrophoneChecker: ObservableObject {
     
     private let audioFilename: URL = {
         let documentPath = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask)[0]
-        let audioFilename = documentPath.appendingPathComponent("recording.m4a")
+        let audioFilename = documentPath.appendingPathComponent("recording.aac")
         return audioFilename
     }()
-    
+
+    private lazy var audioNormaliser: AudioValuePercentageNormaliser = .init()
+
     public init(valueLimit: Int = 3) {
         self.valueLimit = valueLimit
-        self.decibels = [Float](repeating: 0.0, count: valueLimit)
+        self.audioLevels = [Float](repeating: 0.0, count: valueLimit)
         setUpAudioCapture()
     }
     
     /// Checks if there are decibel values available.
     public var hasDecibelValues: Bool {
-        for decibel in decibels {
-            if decibel > Self.minimalDecibelValue {
+        for decibel in audioLevels {
```

Comment:
what does this function do? (and why do we mix decibel and level names?)

Reply:
I renamed it to `isSilent` as this is what it does, it checks if we have values bigger than our `silent` value which is `-50` decibels or 0.